### ISSUE TITLE
JScript compilation error

### ISSUE THREAD

**Issue by JudyJdd** [workshopper/learnyounode#321]
Hi, I am new to node js and new to javascript as well. I installed the learnyounode and try to run it.

It shows a pop-up window when I run "learnyounode" saying "Invalid character". Code is 800A03F6. Could you please help me with that?

Thank you

**Comment by tgfjt** [comment]
Welcome to JS!

Would you give us that you tried? :)

* Windows OS?
* How did you install learnyounode?
* Did you click the file of `learnyounode` to run?

Learnyounode is command line.
You can run like this.
https://youtu.be/c7Nd9Q94cfg

**Comment by JudyJdd** [comment]
Yes, I am in Windows OS. installed learnyounode by "npm install learnyounode -g". I tried to use "sudo" but it is not available in my pc. After installation, I just type in learnyounode, and see the pop-up window.

**Comment by Chrinkus** [comment]
Are you using the windows command line? On my Windows machine I got frustrated with the command prompt so I downloaded the git hub app and power shell. 

Power shell acts like a Mac or Linux command line which is what most of these workshops expect.

**Comment by tgfjt** [comment]
@JudyJdd 
ok. a few questions for you:

* Using Windows version
* `node --version`
* `npm --version`
* `where learnyounode`

It does not seems to be set `NODE_PATH` maybe I think..

**Comment by AndriySikora** [comment]
Hi @tgfjt , I have same problem with learnyounode.
I use Windows 10 OS. I have same error code:  800A03F6.
my node version - 6.91.
npm version - 3.10.9
learyounode is in
D:\Education\IT\JSTernopil\nodeschool\learnyounode\learnyounode\learnyounode.js
C:\Users\Ura\AppData\Roaming\npm\learnyounode

When I use GitBash everything work correct.

Thanks for the help!
